docker/Dockerfile

```diff
@@ -3,21 +3,20 @@ FROM debian:buster-slim
 RUN set -ex; \
     apt-get update -y; \
     apt-get install -y --no-install-recommends \
-        python3 \
-        python3-ldap \
-        python3-pip \
-        python3-psycopg2 \
-        python3-setuptools \
-        python3-yaml \
+        python3=3.7.2-1 \
+        python3-ldap=3.1.0-2 \
+        python3-pip=18.1-5 \
+        python3-psycopg2=2.7.7-1 \
+        python3-setuptools=40.8.0-1 \
+        python3-yaml=3.13-2 \
     ; \
-    apt-get clean; \
     rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*; \
     :
 
 RUN set -ex; \
-    pip3 --no-cache-dir install --no-deps ldap2pg; \
+    pip3 --no-cache-dir install --no-deps ldap2pg==4.18; \
```

Comment:
This is not necessary anymore since Dockerfile now uses DOCKER_TAG as ldap2pg version :tada: 

cf. this test build : of 99999.99999

``` console
Step 4/6 : RUN set -ex; pip3 --no-cache-dir install --no-deps ldap2pg${LDAP2PG_VERSION:+==${LDAP2PG_VERSION}}; ldap2pg --version; :
---> Running in 4c6112247483
+ pip3 --no-cache-dir install --no-deps ldap2pg==99999.99999
Collecting ldap2pg==99999.99999
Could not find a version that satisfies the requirement ldap2pg==99999.99999 (from versions: 1.0a1, 1.0a2, 1.0a3, 1.0b1, 1.0, 2.0a1, 2.0a2, 2.0a3, 2.0, 2.1a1, 3.0a1, 3.0a2, 3.0, 3.1, 3.2, 3.3, 3.4, 4.0, 4.1, 4.2, 4.3, 4.3.1, 4.4, 4.5, 4.6, 4.7, 4.8, 4.9, 4.10, 4.11, 4.12, 4.13b1, 4.13, 4.14, 4.15, 4.16, 4.17, 4.18, 5.0)
No matching distribution found for ldap2pg==99999.99999
```